component-server-parent/component-server-api/src/main/java/org/talend/sdk/component/server/api/ComponentResource.java

```diff
@@ -135,6 +135,23 @@ Response icon(@PathParam("id") @Parameter(name = "id",
             description = "Component icon identifier.",
             in = PATH) String id);
 
+    @GET
+    @Path("searchIcon/{familyId}/{iconKey}")
```

Comment:
I tend to prefer not changing api prefix, rather use `icon`. smtg like `icon/custom/...`.
```suggestion
    @Path("icon/custom/{familyId}/{iconKey}")
```

Reply:
@undx fixed